addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -4,6 +4,7 @@ odoo.define('website.s_image_gallery_options', function (require) {
 var core = require('web.core');
 var weWidgets = require('wysiwyg.widgets');
 var options = require('web_editor.snippets.options');
+const wUtils = require('website.utils');
```

Comment:
Even if it targets 14.0, I think we have to follow the new guidelines, use double quotes :+1: